rasa/core/channels/channel.py

```diff
@@ -158,12 +162,30 @@ def get_metadata(self, request: Request) -> Optional[Dict[Text, Any]]:
         """
         pass
 
+    def _decode_jwt(
+        self, bearer_token: Text, jwt_key: Text, jwt_algorithm: Text
+    ) -> Dict:
+        authorization_header_value = bearer_token.replace(BEARER_TOKEN_PREFIX, "")
+        return jwt.decode(authorization_header_value, jwt_key, algorithms=jwt_algorithm)
+
+    def _decode_bearer_token(
+        self, bearer_token: Text, jwt_key: Text, jwt_algorithm: Text
+    ) -> Optional[Dict]:
+        # noinspection PyBroadException
+        try:
+            return self._decode_jwt(bearer_token, jwt_key, jwt_algorithm)
+        except jwt.exceptions.InvalidSignatureError:
+            logger.error("JWT public key invalid.")
+        except Exception:
+            logger.exception("Failed to decode bearer token.")
+
```

Comment:
why not making this public? both functions don't rely on any object state so I don't see why we should make the class methods. btw, Pycharm also gives a warning because we overrride `_decode_bearer_token` in `RasaChat` with a differrent signature

Reply:
Not sure what I was thinking, agreed making it public and yeah it doesn't rely on state at all